api/v2010/message/create-quickstart/create-quickstart.3.x.js

```diff
@@ -6,7 +6,7 @@ const client = require('twilio')(accountSid, authToken);
 
 client.messages
       .create({
-         body: `Join Earth's mightiest heroes. Like Kevin Bacon.`,
+         body: `Join Earth's mightest heroes. Like Kevin Bacon.`,
```

Comment:
@po5i  this should still read "mightiest" (same with the other `quickstart` samples)